govcd/nsxt_alb_controllers_test.go

```diff
@@ -71,7 +71,10 @@ func (vcd *TestVCD) Test_NsxtAlbController(check *C) {
 	check.Assert(updatedController.NsxtAlbController.Description, Equals, updateControllerDef.Description)
 	check.Assert(updatedController.NsxtAlbController.Url, Equals, updateControllerDef.Url)
 	check.Assert(updatedController.NsxtAlbController.Username, Equals, updateControllerDef.Username)
-	check.Assert(updatedController.NsxtAlbController.LicenseType, Equals, updateControllerDef.LicenseType)
+	// LicenseType has been removed since v37.0
+	if vcd.client.Client.APIVCDMaxVersionIs("< 37.0") {
+		check.Assert(updatedController.NsxtAlbController.LicenseType, Equals, updateControllerDef.LicenseType)
```

Comment:
The funny thing here is that LicenseType is populated in v37.0 as "BASIC", no matter what payload you send. I guess it was done this way for compatibility reasons. That's why this test didn't fail, as the update operation here precisely tested a change from "ENTERPRISE" to "BASIC".

This condition was added to make the test more correct, as you can see it was passing on v37.0 because of coincidence.